test/apiv2/python/rest_api/test_v2_0_0_distribution.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+
+import requests
+from .fixtures import APITestCase
+
+
+class DistributionTestCase(APITestCase):
+    def test_distribution_inspect(self):
+        # Make sure the image exists
+        r = requests.post(self.uri("/images/pull?reference=alpine:latest"), timeout=15)
+        self.assertEqual(r.status_code, 200, r.text)
+
+        r = requests.get(self.podman_url + "/v1.40/distribution/alpine/json")
+        self.assertEqual(r.status_code, 200, r.text)
+
+        result = r.json()
+        self.assertIn("Descriptor", result)
+        self.assertIn("Platforms", result)
+
+        descriptor = result["Descriptor"]
+        self.assertIn("mediaType", descriptor)
+        self.assertIn("digest", descriptor)
+        self.assertIn("size", descriptor)
+
+        for platform in result["Platforms"]:
+            self.assertIn("architecture", platform)
+            self.assertIn("os", platform)
+
+    def test_distribution_inspect_invalid_image(self):
+        r = requests.get(self.podman_url + "/v1.40/distribution/nonexistentimage/json")
+        self.assertIn(r.status_code, (404, 500), r.text)
+
+    def test_distribution_inspect_with_tag(self):
+        r = requests.post(self.uri("/images/pull?reference=postgres:14.7"), timeout=15)
+        self.assertEqual(r.status_code, 200, r.text)
+
+        r = requests.get(self.podman_url + "/v1.40/distribution/postgres:14.7/json")
+        self.assertEqual(r.status_code, 200, r.text)
```

Comment:
never use images not under our control in tests, they can change at any time breaking our CI.
In particular This can never pass CI because we cannot use docker.io, all refernces are redirected to our quay.io/libpod namespace, and even more so we embed all our images  into a local registry running on localhost. So anything that is not in there will fail.

Practically speaking I see nothing here that is different from the alpine example above so maybe just remove this check?